### ISSUE TITLE
OS X: Saved images show up double sized in Preview on retina displays

### ISSUE THREAD

**Issue by kintel** [openscad/openscad#1157]
When exporting as image, or saving images from an animation, they show up double sized in Preview.app. If I instead take a screenshot, it shows up correctly.
The pixel sizes of the two images are the same.
I've tried modifying the DPI of the image from 72 to 144 to match the DPI setting in the screenshot, but that didn't help.

TODO:
* [ ] Figure out how to create a PNG which shows up correctly in Preview.app
* [ ] Convince QImage to save such a PNG

**Comment by gingerbeardman** [comment]
I just came across this issue on my upgraded system.

My model/preview/abnimate/dump workflow was created on a non-retina system so the 2x sized PNGs are a problem for me.

After banging my head trying to:
- set viewport distance
- post-processing resizing
- etc

I found a simple workaround:
- set OpenSCAD to open in "low resolution" mode

<img width="265" alt="Screen shot 2020-08-07 at 18 21 15" src="https://user-images.githubusercontent.com/49612/89671382-c9e63980-d8da-11ea-917f-c6678845c5e1.png">

Of course this means the UI is in low resolution so looks somewhat blurry on a non-retina, but it's a good workaround that can be toggled when needed (just before exporting images).

**Comment by gingerbeardman** [comment]
Please get rid of the bot. They are the death of many projects on GitHub.

**Comment by kintel** [comment]
The bot is only meant to tag issues that have really fallen off the edge, and currently only required annual confirmation that an issue is still warranted. We're also tuning the bot to allow certain issues to stay untouched by the bot. If you have any suggestions in that regard, please let us know.

**Comment by gingerbeardman** [comment]
Get rid of it.

**Comment by kintel** [comment]
Let me rephrase: If you have any _constructive_ suggestions in that regard, please let us know. We're essentially putting more onus on users and reporters of issues to free up limited resources to work on the topics which people are engaging with. The obvious alternative is for more people to volunteer to participate in development work.

**Comment by gingerbeardman** [comment]
More onus how? The issue was reported, the issue was later +1 then zero was done.

How does adding a tag make things better? If the tag enourages the issue to be fixed (which users and reporters cannot do) then that would be great, but all I am seeing is contributors wasting time managing the bot when they could be fixing the very issue that is a problem.

The bot has improved the situation as sum total of zero, in fact with the heat of this discussion I'd say things are worse.

**Comment by kintel** [comment]
It's not clear cut, and we could be more clear. I think one quick action that can be done by users and reporters is verifying that 1. the issue still exists (since issues are often placeholders for upstream bugs, this is important) and 2. reiterating that they are actually still using the software in a way that this issue poses a pain point.

We can do more, we could add a priority tag to rank issues based on feedback etc., we could rename "stalled" to "P4" or "need help". We could improve messaging issued by the bot etc. ..or power users could suggest actions that would improve their lives.

**Comment by gingerbeardman** [comment]
Personally, I think if multiple users put in the effort in the past then the minimum I would expect is for somebody on the project to look into it. Adding tags, or kicking it back to the user is bad faith IMHO. Like, do something about it that involves source code not triaging.

It's the same issue with Apple kicking back bug reports, and that is an equally bad experience.

**Comment by kintel** [comment]
I think, if anything, we should probably be more clear what expectations should be for a project run by a tiny team of volunteers putting in some evening time to make it stay afloat.

**Comment by kintel** [comment]
Let's move any bot-related discussion off this issue. See openscad/trash-can#1
